pkg/ebpf/events_pipeline.go

```diff
@@ -240,22 +240,22 @@ func (t *Tracee) sinkEvents(ctx gocontext.Context, in <-chan *trace.Event) <-cha
 	errc := make(chan error, 1)
 
 	go func() {
-		for event := range in {
```

Comment:
We better keep this for loop on the in channel, so this go routine will be closed when the input channel is closed (one of the concepts in pipelines)

Reply:
Done